pkgs/by-name/dm/dmd/generic.nix

```
@@ -82,6 +82,13 @@ stdenv.mkDerivation (finalAttrs: {
       extraPrefix = "dmd/";
       hash = "sha256-N21mAPfaTo+zGCip4njejasraV5IsWVqlGR5eOdFZZE=";
     })
+  ] ++ lib.optionals (lib.versionOlder version "2.111.0") [
+    (fetchpatch {
+      url = "https://github.com/dlang/dmd/commit/dfe0f34b2aa59e6cf4526c9af8069e08638149bc.patch";
```

This commit is not on a live branch, so I think this URL will eventually expire.

---

A patch file should be included as documented in https://github.com/NixOS/nixpkgs/tree/master/pkgs#patches.

---

@Aleksanaa Wouldn't it be more appropriate to replace it with the hash of the merged commit
```suggestion
      url = "https://github.com/dlang/dmd/commit/fdd25893e0ac04893d6eba8652903d499b7b0dfc.patch";
```